CSS/mainstyle.css

```diff
@@ -279,4 +310,197 @@ ul {
     top: 15px;
     right: 35px;
   }
+
+  .desktop-banner {
+    display: none;
+  }
+
+  .desktop-program-link {
+    display: none;
+  }
+
+  .partners {
+    display: none;
+  }
+
+  .desktop-footer {
+    display: none;
+  }
+}
+
+@media (min-width: 768px) {
+  .mobile-nav {
+    display: none;
+  }
+
+  .desktop-nav-banner {
+    display: flex;
+    background: #3e3e3e;
+    color: white;
+    flex-direction: row;
+    justify-content: end;
+    align-items: center;
+    height: 30px;
+    font-size: 12px;
+    gap: 10px;
+    padding-right: 4em;
+  }
+
+  .desktop-nav {
+    background: white;
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    align-items: center;
+    padding: 0 10em 0 10em;
+    gap: 12em;
+  }
```

Comment:
- [ ] It would be better if you reduce the `gap` you have set for the `nav` as it's causing your page to be less responsive on desktop screen sizes. Please make sure to fix this issue as it would help make your beautiful `website` to be more user-friendly and create a good user experience. **NOTE:** This behavior happens above `768px`. There may be other factors contributing to it so kindly make sure to test your website's responsiveness after implementing the suggested changes to make sure there are no `horizontal` scrolls. See the image below for more details:

<img width="814" alt="Screenshot 2023-05-24 at 18 37 24" src="https://github.com/demesameneshoa/First-Capstone-Project/assets/107192076/c44513ca-e20a-4e66-bf26-4dfde809bcc3">